EcoSoapBank/Model Controller/PaymentController.swift

```diff
@@ -32,7 +32,7 @@ class PaymentController {
     func fetchAllPayments(forPropertyId propertyId: String) -> Future<[Payment], Error> {
 
         Future { promise in
-            self.dataProvider.fetchPayments(forPropertyId: propertyId) { [weak self] result in
+            self.dataProvider.fetchPayments(forPropertyID: propertyId) { [weak self] result in
```

Comment:
And why this one?

Reply:
I did this for consistency. Since the GraphQLController will have this method sitting next to all of the other data provider methods, I wanted it to match the others which use the format `somethingID`.